pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -809,7 +809,12 @@ func (c *Check) StatementMetrics() (int, error) {
 									if err == nil {
 										stepPayload.AccessPredicates = obfuscated.Query
 									} else {
-										stepPayload.AccessPredicates = "obfuscation error"
+										stepPayload.AccessPredicates = "Access obfuscation error"
+										if c.config.ExecutionPlans.LogUnobfuscatedPlans {
+											stepPayload.AccessPredicates = fmt.Sprintf("%s, unobfuscated filter: %s", stepPayload.AccessPredicates, stepRow.AccessPredicates.String)
+										}
+										log.Errorf(stepPayload.FilterPredicates)
```

Comment:
typo? should it be `log.Errorf(stepPayload.AccessPredicates)`?

Reply:
Yes, thx!. Fixed now.